test/user.spec.js

```diff
@@ -152,3 +153,61 @@ describe('POST /users/sessions  - signIn user', () => {
     });
   });
 });
+
+describe('GET /users - list of users', () => {
+  const token = authenticationService.generateToken(userData);
+  const anotherUser = {
+    email: 'kevin@wolox.com.ar',
+    name: 'kevin',
+    lastName: 'perez',
+    password: 'asdasdasd'
+  };
+  it('should response with 200 code', done => {
+    createUserModel(userData).then(() => {
+      console.log('uno');
+      createUserModel(anotherUser).then(() => {
+        request(app)
+          .get('/users')
+          .set('Authorization', `Bearer ${token}`)
+          .then(res => {
+            expect(res.status).toBe(200);
+            expect(res.body.users.totalPages).toBe(1);
+            expect(res.body.users.users.count).toBe(2);
+            const arr = res.body.users.users.rows;
+            expect(arr.slice(1).every((item, i) => arr[i].email < item.email)).toBe(true);
+            done();
+          });
+      });
+    });
+  });
+
+  it('should response with 200 code', done => {
+    createUserModel(userData).then(() => {
+      createUserModel(anotherUser).then(() => {
+        request(app)
+          .get('/users?limit=20&page=1&orderBy=name&order=desc')
+          .set('Authorization', `Bearer ${token}`)
+          .then(res => {
+            expect(res.status).toBe(200);
+            expect(res.body.users.totalPages).toBe(1);
+            expect(res.body.users.users.count).toBe(2);
+            const arr = res.body.users.users.rows;
+            expect(arr.slice(1).every((item, i) => arr[i].email > item.email)).toBe(true);
```

Comment:
Why are you removing first element ? (You can create a function for this if its repeated several times)